frontend/asset_mapper.rst

```diff
@@ -402,6 +402,8 @@ from inside ``app.js``:
     // things on "window" become global variables
     window.$ = $;
 
+.. _asset-mapper-handling-css:
```

Comment:
I'm relatively new to writing RST. If I'm not mistaken this is the correct way to link to another section in the same document, is that correct?

Reply:
Yes it's right :)